scripts/indexd_utils.py

```diff
@@ -1,12 +1,12 @@
-from cdislogging import get_logger
+import logging as logger
 from errors import APIError, UserError
 import os
 import utils
 from settings import PROJECT_ACL
 from urlparse import urlparse
 
 
-logger = get_logger("IndexdUtils")
+logger.basicConfig(level=logger.INFO, format="%(asctime)s %(message)s")
```

Comment:
note: using logging instead of cdislogging because cdislogging is not on pypi and scripts running in google cloud must only have pypi dependencies